nixos/modules/services/mail/roundcube.nix

```diff
@@ -148,21 +153,20 @@ in
         ${cfg.hostName} = {
           forceSSL = mkDefault true;
           enableACME = mkDefault true;
+          root = cfg.package;
           locations."/" = {
-            root = cfg.package;
             index = "index.php";
             extraConfig = ''
-              location ~* \.php(/|$) {
-                fastcgi_split_path_info ^(.+\.php)(/.+)$;
-                fastcgi_pass unix:${fpm.socket};
-
-                fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
-                fastcgi_param PATH_INFO       $fastcgi_path_info;
-
-                include ${config.services.nginx.package}/conf/fastcgi_params;
-                include ${pkgs.nginx}/conf/fastcgi.conf;
-              }
-            '';
+              add_header Cache-Control 'public, max-age=604800, must-revalidate';
```

Comment:
Is this from any upstream source? If not, have you checked that there are no things that shouldn't be cached, but are handled in this location block?

Reply:
Yes, tested on my server, all dynamic requests fall into the adjacent block.

Reply:
Yeah, seems like dynamic stuff goes through index.php (and thus the PHP location), everything else seems cacheable :+1: